README.md

```diff
@@ -41,6 +41,8 @@ v3.0 generates plugins that are compatible with versions of SonarQube from v7.9.
 
 v3.1 generates plugins that are compatible with versions of SonarQube from v7.9.6 (tested with the latest available version at time of release i.e. v9.7).
 
+v4.0 generates plugins that are compatible with versions of SonarQube from v9.9 (tested with the latest available version at time of release i.e. v10.5.1).
```

Comment:
Should we add a link to the template repository somewhere in the README?

Reply:
No, as it will likely move.